releasenotes/notes/gradient-based-adapt-vqe-aa4e89f44c5aa69e.yaml

```diff
@@ -0,0 +1,21 @@
+---
+features:
+  - |
+    Added Qiskit Terra's Gradient Functionality to calculate gradients in AdaptVQE. You can choose
+    the gradient method and input it while calling ``AdaptVQE``.
+    for example::
+
+      from qiskit.opflow.gradients import Gradient
+      grad = Gradient(grad_method="lin_comb", epsilon=1.0)
+      calc = AdaptVQE(self.qubit_converter, solver, gradient=grad)
```

Comment:
Would it be nice if this is a working example since this is what users will see in the releasenotes? 🙂